src/stirling/source_connectors/socket_tracer/socket_trace_connector.cc

```diff
@@ -332,10 +332,13 @@ const auto kProbeSpecs = MakeArray<bpf_tools::KProbeSpec>(
      {"close", ProbeType::kReturn, "syscall__probe_ret_close"},
      {"mmap", ProbeType::kEntry, "syscall__probe_entry_mmap"},
      {"sock_alloc", ProbeType::kReturn, "probe_ret_sock_alloc", /*is_syscall*/ false},
-     {"security_socket_sendmsg", ProbeType::kEntry, "probe_entry_security_socket_sendmsg",
-      /*is_syscall*/ false},
-     {"security_socket_recvmsg", ProbeType::kEntry, "probe_entry_security_socket_recvmsg",
-      /*is_syscall*/ false}});
+     {"security_socket_sendmsg", ProbeType::kEntry, "probe_entry_socket_sendmsg",
+     /*is_syscall*/ false, /* is_optional */ false,
+     new bpf_tools::KProbeSpec{"sock_sendmesg", ProbeType::kEntry, "probe_entry_socket_sendmsg", false, true}},
+     {"security_socket_recvmsg", ProbeType::kEntry, "probe_entry_socket_recvmsg",
+     /*is_syscall*/ false, /* is_optional */ false,
+     new bpf_tools::KProbeSpec{"sock_recvmsg", ProbeType::kEntry, "probe_entry_socket_recvmsg", false, true}},
+  });
```

Comment:
There are also some linting errors here that can be identified by running `arc lint` (assuming your development environment is setup).